b2b-sample/apps/business-app/components/sections/storyHomes/easyMeetingsHome.tsx

```diff
@@ -71,7 +89,7 @@ export default function Home(prop) {
 
                     <HomeComponent
                         scope={ session.scope }
-                        sideNavData = { sideNavData }
+                        sideNavData={ sideNavData }
                         activeKeySideNav={ activeKeySideNav }
```

Comment:
Shall we add spaces before and after = in other lines also to make them consistant

Reply:
For props we do not use space before and after a `=`